### DISCUSSION TITLE
translating

### DISCUSSION THREAD

**Discussion by RoDanny2021** [cdgriffith/FastFlix#469] [General]
How do I translate the interface into romanian languages?

**Comment by cdgriffith** [comment]
Hi @RoDanny2021, thanks for the idea! I am adding the automatic translations in the branch `Romanian` https://github.com/cdgriffith/FastFlix/blob/Romanian/fastflix/data/languages.yaml   

It is the file `fastflix/data/languages.yaml` if you want to make any fixes you can either fork the repo and do a pull request, or can just download that file and make changes to it, and attach it back here in a zip file!

**Reply by RoDanny2021** [reply]
Thank you. I see it is already translated.
In the software, however, I cannot choose the Romanian language, why?
Only English, Chinese, Italian, French, Spanish, German, Russian, Japanese, Portuguese , Swedish, Polish ...why?

**Reply by cdgriffith** [reply]
It's not currently released. I just went and ran those translations last night on your request, and they are not in the main build yet. 

Wanted to give you the chance to correct any you see that are horribly wrong before putting them into the software. But if you are fine with auto translations, I can just add them in to the main build later this week!

**Comment by RoDanny2021** [comment]
Hello,

I checked the jumps and they seem quite well translated.

Danny

**Comment by cdgriffith** [comment]
Added in 5.5.2! https://github.com/cdgriffith/FastFlix/releases/tag/5.5.2

**Comment by RoDanny2021** [comment]
Hello,

It’s a problem....

 

rou is not the abbreviation for the Romanian language but ro

 

https://registry-page.isdcf.com/languages/

 

Romanian – ro  RO 

 

 

Thanks for mentioning me:

 



 

and yet in version 5.5.2 you cannot select the Romanian language…

I have attached the modified yaml file changing "ro" to "ro".

 

Best,

Danny

**Comment by cdgriffith** [comment]
I use  ISO 639 codes not DCNC  https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

However yes, totally forgot to add it to the dropdown in the setting! Fixing

**Comment by RoDanny2021** [comment]
It's not a problem. Next version….

Ok but in https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes for Romania is „ro” or  „ron” but usually is „ro” like languages in DVD/Blu-ray subtitles....

 

I make change in “yaml” file and change “rou”  with “ro” .

But I don’t know how can put on github….

 

I’m novice….. sorry

Best

Danny

**Comment by cdgriffith** [comment]
Good catch, yes needed to be ron, was just the translation stuff I was using wanted rou, editing for in the langague file! 

Should work now https://github.com/cdgriffith/FastFlix/releases/tag/5.5.3

**Comment by RoDanny2021** [comment]
Hello again...... 

After I chose the Romanian language and started to check, I found many strange things translated

The problem is that even in English it sounds strange.

I don't understand what you wanted to say in English here....

“Slow is highest personal recommenced, as past that is much smaller gains”

 

 

I will try to go through these days to see what sounds strange. It's simpler here on the interface than in the yaml file     

 

Where I don't understand I will ask you                

                

I think the above sentence is wrong in English

Best,

Danny

**Comment by RoDanny2021** [comment]
I think u want say:

 

Adjusts encoder settings to trade off compression efficiency against encoding speed.

This establishes your default encoder settings.

 

Tunes, profiles, levels and advanced option string will be applied to this.

 

You should generally set this option to the slowest you can bear since slower settings will result better quality or smaller files.

 

I found this explication on other encoder......
